scripts/runners/nrf_common_next.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+'''
+Import nrf_common.py and override/include additional
+functionality required for nRF7120
+'''
+
+import os
+import sys
+
+FILE_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(f"{FILE_PATH}/../../../zephyr/scripts/west_commands/runners")
```

Comment:
can this not use ZEPHYR_BASE? Not sure if it would be set in environment or how you get it from python